stubs/pysftp/METADATA.toml

```
@@ -0,0 +1,3 @@
+version = "0.1"
```

This indicates the oldest version that these stubs support, but pysftp 0.1 doesn't even seem to exist (https://pypi.org/project/pysftp/#history).
```suggestion
version = "0.2"
```